.rubocop.yml

```diff
@@ -26,23 +23,8 @@ Metrics/MethodLength:
   Exclude:
     - "spec/features/*"
 
-Style/ClassAndModuleChildren:
-  Enabled: false
-
-Style/Documentation:
-  Enabled: false
-
-Style/DoubleNegation:
-  Enabled: false
-
 Style/StringLiterals:
   Enabled: false
```

Comment:
We have enforced double_quotes here in the main coding style file. Shouldn't we let that be enabled here?